File: 2015-2016/Contests/Official/4) ECOO/ECOOCS_2015/Round 1/Solutions/Problem 0.cpp
#include <iostream>

using namespace std;

int main() {
    freopen("DATA01.txt", "r", stdin);

    int a,b,c,d,e;
    for (int i = 0; i < 5; i++)
    {
	cin >> a >> b >> c >> d >> e;
	cout << (200*a+100*b+25*c+10*d+5*e) << endl;
    }
}

File: 2014-2015/Contests/ECOO/Round 1/Commented Solutions/Problem 0.py
# file input
import sys
sys.stdin = open("DATA01.txt", "r")

# 5 test cases
for test in range(5):
    
    # input toonies, loonies, quarters, dimes, nickels
    a, b, c, d, e = map(int, raw_input().strip().split())

    # print number of cents
    print ( (200 * a) + (100 * b) + (25 * c) + (10 * d) + (5 * e) )
